app/rtc_compare.py

```diff
@@ -629,15 +646,21 @@ def compare_rtc_s1_products(file_1, file_2):
                   f' ({image_1.dtype}) vs. ({image_2.dtype})\n')
             return False
 
-        flag_bands_are_equal = np.allclose(
+        failed_pixel_index = ~np.isclose(
             image_1, image_2, atol=RTC_S1_PRODUCTS_ERROR_ABS_TOLERANCE,
             rtol=RTC_S1_PRODUCTS_ERROR_REL_TOLERANCE, equal_nan=True)
+        failed_pixel_ratio = failed_pixel_index.sum() / np.isfinite(image_1 + image_2).sum()
```

Comment:
```suggestion
        number_valid_pixels = np.logical_or(np.isfinite(image_1),
            np.isfinite(image_2)).sum()
        failed_pixel_ratio = failed_pixel_index.sum() / number_valid_pixels
```

Reply:
Uhm.. I'm thinking that if we use `image_1 + image_2` and one of the images is (almost) full of NaNs, we may not be able to do a good comparison. How about something like the suggested code above?